float32 th1
float32 th2
float32 th3
float32 d1
float32 d2
float32 d3
bool cone_detected
bool obj_detected
